module/zfs/dmu_send.c

```diff
@@ -1084,10 +1131,12 @@ dmu_send_impl(void *tag, dsl_pool_t *dp, dsl_dataset_t *to_ds,
 	dsp->dsa_os = os;
 	dsp->dsa_off = off;
 	dsp->dsa_toguid = dsl_dataset_phys(to_ds)->ds_guid;
+	dsp->dsa_fromtxg = fromtxg;
 	dsp->dsa_pending_op = PENDING_NONE;
 	dsp->dsa_featureflags = featureflags;
 	dsp->dsa_resume_object = resumeobj;
 	dsp->dsa_resume_offset = resumeoff;
+	dsp->dsa_spill = zfs_send_set_spill_block_bit;
```

Comment:
why bother with this, vs. using zfs_send_set_spill_block_bit in place of dsa_spill?

Reply:
I wanted to ensure this value would not change while creating the send stream.  Since `zfs_send_set_spill_block_bit` is a global and it could be (but currently isn't) available as a module option.  I primarily added this flag to make my testing easier.  Since I can't think of a reason we'd want to disable this, I'll remove it and `zfs_send_set_spill_block_bit`.